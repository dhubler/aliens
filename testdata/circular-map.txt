Boston north=Bangor
Bangor north=Trenton 
Trenton north=Boston
