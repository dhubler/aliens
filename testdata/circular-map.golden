Bangor north=Trenton south=Boston
Boston north=Bangor south=Trenton
Trenton north=Boston south=Bangor
